src/process_mcp_repos.py

```diff
@@ -14,17 +14,15 @@
 
 # Import the local functions
 from .github import get_github_client, enable_ghas_features, check_dependabot_config, clone_or_update_repo, extract_repo_owner_name, get_repository_properties, handle_github_api_error, list_all_repositories_for_org, list_all_repository_properties_for_org, show_rate_limit, update_repository_properties
+from .constants import TARGET_ORG, MCP_AGENTS_HUB_REPO_URL, LOCAL_REPO_PATH, SERVER_FILES_DIR_IN_REPO
 
 # Configuration
 logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
 logging.getLogger("githubkit").setLevel(logging.WARNING)  # Reduce verbosity from githubkit
 load_dotenv()  # Load environment variables from .env file
 
-# Constants
-MCP_AGENTS_HUB_REPO_URL = "https://github.com/mcp-agents-ai/mcp-agents-hub.git"
-LOCAL_REPO_PATH = Path("./cloned_mcp_agents_hub")
-server_files_from_loader_DIR_IN_REPO = Path("server/src/data/split")
-TARGET_ORG = "mcp-research"  # The organization to fork into
+# Convert the SERVER_FILES_DIR_IN_REPO string to a Path object
+server_files_from_loader_DIR_IN_REPO = Path(SERVER_FILES_DIR_IN_REPO)
```

Comment:
Let's move this path conversion into the constants file, as we use it as a path everywhere, so better to handle this centrally.